inventree_brother/brother_plugin.py

```diff
@@ -57,6 +80,8 @@ class BrotherLabelPlugin(LabelPrintingMixin, SettingsMixin, InvenTreePlugin):
     SLUG = "brother"
     TITLE = "Brother Label Printer"
 
+    PrintingOptionsSerializer = BrotherLabelSerializer
```

Comment:
Comment: Yep, that's another way of defining the options I haven't thought of, but Program wise they are exactly considered the same.